README.md

```diff
@@ -0,0 +1,21 @@
+# mlflow-elasticsearchstore
+
+Mlflow plugin to use Elaticsearch as Tracking Store in Mlflow. To use this plugin you need a running instance of Elasticsearch 6.8.
+
+## Installation
+
+In a python environment :
+
+```bash
+$ git clone git clone https://github.com/criteo/mlflow-elasticsearchstore.git
+$ cd mlflow-elasticsearch
+$ pip install .
+```
```

Comment:
You can specify that this is for development, for a common usage we will publish a package on pypi so `pip install mlflow-elasticsearchstore` should be sufficient (do not put it for now since we haven't published the package yet)